tests/Request.test.ts

```diff
@@ -45,12 +49,13 @@ describe('Request', () => {
          expect(new Request(app, _.extend({}, albRequest(), { httpMethod: 'get' }), handlerContext()).method).to.strictlyEqual('GET');
          expect(new Request(app, _.extend({}, albRequest(), { httpMethod: 'PoSt' }), handlerContext()).method).to.strictlyEqual('POST');
 
-         // make sure that undefined values don't break it:
-         let evt2: RequestEvent = albRequest();
+         // TODO - this doesn't seem possible without encountering: error TS2790: The operand of a 'delete' operator must be optional.
+         // // make sure that undefined values don't break it:
+         // let evt2: RequestEvent = albRequest();
 
-         delete evt2.httpMethod;
-         expect(evt2.httpMethod).to.strictlyEqual(undefined);
-         expect(new Request(app, evt2, handlerContext()).method).to.strictlyEqual('');
+         // delete evt2.httpMethod;
+         // expect(evt2.httpMethod).to.strictlyEqual(undefined);
+         // expect(new Request(app, evt2, handlerContext()).method).to.strictlyEqual('');
```

Comment:
Flagging this TODO for feedback. I think that this no longer works with the updated type definitions and needs to be removed.